haystack/preview/components/__init__.py

```diff
@@ -0,0 +1,2 @@
+from haystack.preview.components.prompt.prompt import Prompt
```

Comment:
I am assuming somebody would have already pointed this out but this import sounds very weird as a user.